fabtests/common/shared.c

```diff
@@ -3565,6 +3645,41 @@ int ft_check_buf(void *buf, size_t size)
 	return ret;
 }
 
+int ft_rma_poll_buf(void *buf, int iter, size_t size)
+{
+	volatile char *recv_data;
+	struct timespec a, b;
+
+	if (opts.iface != FI_HMEM_SYSTEM) {
+		/* Not supported */
+		FT_ERR("FI_HMEM not supported for writedata latency test");
```

Comment:
`write` or `writedata`? AFAICT writedata uses CQ to poll completion

Reply:
`write` is correct. Will fix.